demo.sh

```diff
@@ -0,0 +1,6 @@
+l#!/bin/bash -e
+
+cd ./keycloak
+wget https://github.com/orchestracities/keycloak-scripts/releases/download/v0.0.4/oc-custom.jar -O oc-custom.jar
+cd ..
+docker-compose up
```

Comment:
you should include as well few "tests" to:
1. load relevant policies in the api
2. generate token from keycloak
3. send requests ok and ko to the PEP